client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx

```diff
@@ -203,11 +203,11 @@ const InvestigationTable: React.FC = (): JSX.Element => {
     }
 
     const getTableCell = (cellName: string, indexedRow: { [T in keyof typeof TableHeadersNames]: any }) => {
-        const wasInvestigationFetchedByGroup = indexedRow.groupId && !indexedRow.canFetchGroup;
+        const wasInvestigationFetchedByGroup = Boolean(indexedRow.groupId) && !indexedRow.canFetchGroup;
         switch (cellName) {
             case TableHeadersNames.color:
                 return (
-                    indexedRow.groupId ?
+                    Boolean(indexedRow.groupId) ?
```

Comment:
I don't think you need the Boolean

Reply:
its not necessary but its more typescript-_y_